### DISCUSSION TITLE
run  my model with the object-detector-python

### DISCUSSION THREAD

**Discussion by MenahemOwlytics** [AxisCommunications/acap-computer-vision-sdk-examples#167] [Q&A] [answered]
I am trying to run the object-detector-python example with my model to detect specific objects

There is a major difference between the models in terms of input size. The example MPDEL received input of 300x300*3 while my model received input of 640x640*3 

I got the following error when I tried to run it:

 Could not run job: Could only read 1228800 out of 4915200 bytes from file descriptor

How can I change the maximum number of bytes that can be read?

Thanks

**Comment by Corallo** [comment]
Hi @MenahemOwlytics 
You are probably using a wrong data type, make sure to set the input as `np.uint8` or `np.int8` depending on the model you are using

**Reply by MenahemOwlytics** [reply]
@Corallo, thanks for the quick response

My model input is:
`[{'name': 'inputs_0', 'index': 0, 'shape': array([  1, 640, 640,   3], dtype=int32), 'shape_signature': array([  1, 640, 640,   3], dtype=int32), 'dtype': <class 'numpy.float32'>, 'quantization': (0.0, 0), 'quantization_parameters': {'scales': array([], dtype=float32), 'zero_points': array([], dtype=int32), 'quantized_dimension': 0}, 'sparsity_parameters': {}}]`

So i think the input type is not the problem

Nevertheless, I added:
`image = np.array(image, dtype=np.unit32)`
or `image = np.array(image, dtype=np.unit32)`

but it not success 

this is the relevant code:
```
  def detect(self, image):
        image = np.expand_dims(image, axis=0)
        image = np.array(image, dtype=np.unit32)
        image = image.astype(np.uint8)
        success, result = self.inference_client.infer({'data': image}, self.model_path)
```

Once again, thank you

**Reply by Corallo** [reply, marked as answer]
I made a mistake before, the error is actually telling you that you are only providing one quarter of the amount of bytes needed by the model.
Try to remove the line:
`image = image.astype(np.uint8)`
However, I am not sure if this model can run on DLPU, it should work only with int8 models, but maybe it is still worth to try.
